symphony-bdk-legacy/symphony-api-client-java/src/test/java/utils/CertificateUtilsTest.java

```diff
@@ -0,0 +1,55 @@
+package utils;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.security.cert.CertificateException;
+import java.security.cert.X509Certificate;
+
+import static org.junit.Assert.*;
+
+public class CertificateUtilsTest {
+
+  private static final Logger logger = LoggerFactory.getLogger(CertificateUtils.class);
+
+  @Test
+  public void testParseX509Certificate() throws IOException {
+    final String currentPath = System.getProperty("user.dir");
+    final String megabotPath = currentPath + "/src/test/resources/megabot-ca.crt";
+    final String textCertificate = new String(Files.readAllBytes(Paths.get(megabotPath)));
+    assertNotNull(textCertificate);
+
+    try {
+
+      final X509Certificate x509Certificate = CertificateUtils.parseX509Certificate(textCertificate);
+      x509Certificate.checkValidity();
+
+      assertTrue(true);
+
+    } catch (final CertificateException e) {
+      fail();
```

Comment:
You can remove the try/catch as the test will fail if there is an exception thrown.

Reply:
I can't remove this try/catch. An error is thrown :

![try_cath_certificateUtils](https://user-images.githubusercontent.com/68892288/95573131-44dae600-0a2b-11eb-89c1-c046788a76da.png)

So I don't understand how to do

Reply:
@Haiddi-Avencod your test method should `throw IOException, CertificateException`

Reply:
Ok